mindmeld/system_entity_recognizer.py

```diff
@@ -13,15 +13,19 @@
 import json
 import logging
 import sys
-
+import os
 import requests
 
 from mindmeld.components._config import (
     get_system_entity_url_config,
     is_duckling_configured,
 )
+from mindmeld.exceptions import MindMeldError
 
 NO_RESPONSE_CODE = -1
+SYS_ENTITY_REQUEST_TIMEOUT = float(os.environ.get("MM_SYS_ENTITY_REQUEST_TIMEOUT", 1.0))
+if SYS_ENTITY_REQUEST_TIMEOUT <= 0.0:
+    raise MindMeldError("SYS_ENTITY_REQUEST_TIMEOUT value has to be greater than 0.0 seconds")
```

Comment:
```suggestion
    raise MindMeldError("SYS_ENTITY_REQUEST_TIMEOUT value has to be greater than 0 seconds")
```

Reply:
I think representing the error as a double is more clear to the end user